connectors/db/vector.py

```diff
@@ -164,7 +226,8 @@ class VectorStoreInterface:
     def __init__(self):
         self.store = None
         self.vector_chunk_size = 2000
-        self.vector_chunk_overlap = 0
+        self.vector_min_chunk_size = 200
+        self.vector_chunk_overlap = 200
```

Comment:
I am thinking for simplicity, should we just get rid of the concept of "min chunk size" and just re-combine chunks up to a "max chunk size"?

```suggestion
        self.max_chunk_size = 2300
        self.chunk_overlap = 200
```

Reply:
I think this would work. Something feels weird about it to me, something I can't quite put my finger on. It feels like we'd end up with a lot more small chunks because we'd be trying to combine chunks that frequently would exceed max_chunk_size and thus we'd skip them. I could try it and see what it does to the chunk size distribution.